Makefile

```diff
@@ -7,6 +7,8 @@ DASHBOARD_CONTAINER_NAME=dashboard-proxy
 DASHBOARD_IMAGE_NAME=jupyter-incubator/$(DASHBOARD_CONTAINER_NAME)
 KG_IMAGE_NAME=jupyter-incubator/kernel-gateway-extras
 KG_CONTAINER_NAME=kernel-gateway
+HTTP_PORT=3000
+HTTPS_PORT=3001
```

Comment:
Yes, in make. Also, if they're only used in one target, put them on that target only. (I don't have the full source in front of me at the moment to check.)